library/src/main/camera2/com/google/android/cameraview/Camera2.java

```diff
@@ -835,6 +797,37 @@ private void configureTransform() {
                             }, 0,
                     4);
         }
+
+        // If the aspect ratio of the TextureView surface doesn't match the ratio, we need to crop the surface view
```

Comment:
This is an important bit as well. When the aspect ratio of the preview is not the same as the CameraView we need to apply transformations to the surface so that the preview content doesn't look squashed or moved.